static/app/views/explore/components/table.tsx

```diff
@@ -74,10 +76,13 @@ export function useTableStyles({
 }
 
 export const TableBody = GridBody;
+export const TableRow = GridRow;
 export const TableBodyCell = GridBodyCell;
+
 export const TableHead = GridHead;
-export const TableHeadCell = GridHeadCell;
 export const TableHeader = Header;
 export const TableHeaderActions = HeaderButtonContainer;
 export const TableHeaderTitle = HeaderTitle;
-export const TableRow = GridRow;
+export const TableHeadCell = styled(GridHeadCell)<{align?: Alignments}>`
+  justify-content: ${({align}) => (align ? align : 'right')};
+`;
```

Comment:
Why does this default to `right` when most columns are left aligned? I wouldve expected something more closer to

```typescript
${p => p.align && `justify-content: ${p.align};`}
```

Reply:
Agree, yes my bad I'll push the change